layouts/partials/home/cncf-status.html

```diff
@@ -0,0 +1,15 @@
+{{ $title := site.Title }}
+<section class="container-xl">
+  <div class="row">
+    <div class="col-12">
+      <hr>
+      <p>
+        <a href="https://cncf.io">
+          <img src="img/logos/cncf-color.png" width="200">
+        </a>
+        </br>
+        {{ $title }} is a Cloud Native Computing Foundation sandbox project.
```

Comment:
```suggestion
        </p>
        <p>
        {{ $title }} is a Cloud Native Computing Foundation sandbox project.
```

Rather than use br, wrap them both in separate <p> tags for now.